src/Pages/Form/Components/PatientInformationSection.tsx

```diff
@@ -0,0 +1,265 @@
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#FFF",
+      contrastText: "#3A3A3D",
+    },
+    secondary: {
+      main: "#EF5DA8",
+      contrastText: "#FFF",
+    },
+  },
+});
+
+const useStyle = makeStyles((theme) =>
+  createStyles({
+    root: {
+      width: "100%",
+      marginLeft: "32px",
+    },
+    picker: {
+      marginTop: "34px",
+    },
+    textField: {
+      width: "80%",
+    },
+    textTitle: {
+      marginTop: "40px",
+    },
+    boxMale: {
+      width: "165px",
+      textAlign: "center",
+    },
+    boxFemale: {
+      width: "190px",
+      textAlign: "center",
+    },
+    button: {
+      "&.Mui-selected": {
+        backgroundColor: "#EF5DA8",
+        color: "#FFF",
+        pointerEvents: "none",
+      },
+    },
+    buttonUpload: {
+      textTransform: "none",
+      marginTop: "16px",
+      color: "#FFF",
+      backgroundColor: "#3A3A3D",
+      boxShadow: "0px 2px 5px rgba(0, 0, 0, 0.1)",
+      borderRadius: "20px",
+      width: "170px",
+      height: "40px",
+    },
+  })
+);
+
+const GenderStyledToggleButtonGroup = withStyles((theme) => ({
+  grouped: {
+    margin: theme.spacing(3),
+    boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+    width: "150px",
+    height: "95px",
+    backgroundColor: "#FFF",
+    border: "none",
+    "&:not(:first-child)": {
+      borderRadius: "20px",
+    },
+    "&:first-child": {
+      borderRadius: "20px",
+    },
+  },
+}))(ToggleButtonGroup);
+
+const OnsetStyledToggleButtonGroup = withStyles((theme) => ({
+  grouped: {
+    textTransform: "none",
+    margin: theme.spacing(3),
+    boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+    width: "220px",
+    height: "80px",
+    backgroundColor: "#FFF",
+    color: "#000",
+    border: "none",
+    "&:not(:first-child)": {
+      borderRadius: "20px",
+    },
+    "&:first-child": {
+      borderRadius: "20px",
+    },
+  },
+}))(ToggleButtonGroup);
+
+const PatientInformationSection: React.FC = () => {
+  const classes = useStyle();
+
+  //* Gender Button
+  const [gender, setGender] = React.useState<string | null>("left");
+  const handleGender = (
+    event: React.MouseEvent<HTMLElement>,
+    newGender: string | null
+  ) => {
+    setGender(newGender);
+  };
+
+  //* Onset Button
+  const [onset, setOnset] = React.useState<string | null>("left");
+  const handleOnset = (
+    event: React.MouseEvent<HTMLElement>,
+    newOnset: string | null
+  ) => {
+    setOnset(newOnset);
+  };
+
+  //* Date and Time Picker
+  const [selectedDate, setSelectedDate] = React.useState<Date | null>(
+    new Date()
+  );
+  const handleDateChange = (date: Date | null) => {
+    setSelectedDate(date);
+  };
+
+  return (
+    <Box className={classes.root}>
+      <ThemeProvider theme={theme}>
+        {/* Patient - Text Field */}
+        <Grid container spacing={6}>
+          <Grid item xs={4}>
```

Comment:
In my opinion, TextField is too long. But if you think that is appropriate ,I'm ok.